org.raceintospace.Raceintospace.yml

```diff
@@ -0,0 +1,89 @@
+# vim: sts=4:
+app-id: org.raceintospace.Raceintospace
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: raceintospace
+#rename-icon: raceintospace
+rename-desktop-file: raceintospace.desktop
+rename-appdata-file: raceintospace.appdata.xml
+modules:
+    - name: sdl
```

Comment:
`sdl` is already there in shared-modules: https://github.com/flathub/shared-modules/tree/master/SDL

Reply:
Sounds nice. However, it is possible to include it into Yaml formatted file? If so, how? I haven't found any reference on that repo. It expect json format, which was not mentioned by tutorial I was following. Did I choose wrong format? Is there up-to-date documentation how shared modules could be used?

Reply:
Okay, found it is simple. It would be nice to have YAML example how to do it somewhere, instead just guessing.